source/_posts/2021-03-12-community-highlights.markdown

```diff
@@ -0,0 +1,123 @@
+---
+title: "Community Highlights: 12th edition"
+description: ""
+date: 2021-03-12 00:00:00
+date_formatted: "March 12, 2021"
+author: Klaas Schoute
+author_twitter: klaasnicolaas
+categories: Community
+og_image: /images/blog/2021-03-12-community-highlights/social.png
+---
+
+The 12th edition of the Home Assistant Community Highlights! Some interesting
+things popped up around our community, we thought was worth sharing.
+
+Do you want to share something for the next edition?
+Information on [how to share](#got-a-tip-for-the-next-edition).
+
+./Klaas
+
+## Blueprint of the week
+
+Do you have a number of domain names and would you like to track when an SSL
+certificate expires? Then you could try [bkbilly's](https://community.home-assistant.io/u/bkbilly)
+blueprint that automatically sends you a notification when a sensor exceeds your set threshold value.
+[Try it out!](https://community.home-assistant.io/t/certificate-expiration/280125)
+
+<div style="margin:0 auto; text-align:center">
+    <a href="https://community.home-assistant.io/t/certificate-expiration/280125" target="_blank">
+        <img
+            src='/images/blog/2021-03-12-community-highlights/blueprint.png'
+            alt="How this blueprint works"
+            style='border: 0;box-shadow: none;width:90%;'
+        />
+    </a>
+</div>
+
+Also take a look at the [Blueprint Exchange][blueprints] for more inspiration!
+
+## Birthday cake
+
+Happy birthday to [adlawton](https://www.reddit.com/user/adlawton/)! 🥳 For his birthday,
+his wife baked this great Home Assistant cake and also gave him a [Home Assistant Blue][blue].
```

Comment:
Blue is a link within this website, we should not use a full URL in those cases.

```suggestion
his wife baked this great Home Assistant cake and also gave him a [Home Assistant Blue](/blue).
```